apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.ts

```diff
@@ -30,27 +36,48 @@ router.get(
       { course_instance_id: res.locals.course_instance.id },
       z.string(),
     );
-
+    const assessmentSets = await sqldb.queryRows(
+      sql.select_assessment_sets,
+      { course_id: res.locals.course.id },
+      AssessmentSetSchema,
+    );
+    const modules = await sqldb.queryRows(
```

Comment:
Let's be consistent with naming. If we use `assessmentSets` above, let's use `assessmentModules` here.

Reply:
Updated to use `assessmentModules` in [9173a63](https://github.com/PrairieLearn/PrairieLearn/pull/10677/commits/9173a63860cec149d3796686882b0ec384a3dc16).